.github/scripts/generate_binary_build_matrix.py

```diff
@@ -88,17 +88,26 @@
         "nvidia-cufile-cu12==1.13.0.11; platform_system == 'Linux' and platform_machine == 'x86_64'"
     ),
     "xpu": (
-        "intel-cmplr-lib-rt==2025.0.4; platform_system == 'Linux' | "
-        "intel-cmplr-lib-ur==2025.0.4; platform_system == 'Linux' | "
-        "intel-cmplr-lic-rt==2025.0.4; platform_system == 'Linux' | "
-        "intel-sycl-rt==2025.0.4; platform_system == 'Linux' | "
-        "intel-cmplr-lib-rt==2025.0.5; platform_system == 'Windows' | "
-        "intel-cmplr-lib-ur==2025.0.5; platform_system == 'Windows' | "
-        "intel-cmplr-lic-rt==2025.0.5; platform_system == 'Windows' | "
-        "intel-sycl-rt==2025.0.5; platform_system == 'Windows' | "
-        "tcmlib==1.2.0 | "
-        "umf==0.9.1 | "
-        "intel-pti==0.10.1"
+        "intel-cmplr-lib-rt==2025.1.1 | "
+        "intel-cmplr-lib-ur==2025.1.1 | "
+        "intel-cmplr-lic-rt==2025.1.1 | "
+        "intel-sycl-rt==2025.1.1 | "
+        "oneccl-devel==2021.15.1; platform_system == 'Linux' and platform_machine == 'x86_64' | "
+        "oneccl==2021.15.1; platform_system == 'Linux' and platform_machine == 'x86_64' | "
+        "impi-rt==2021.15.0; platform_system == 'Linux' and platform_machine == 'x86_64' | "
```

Comment:
For reviewers' information - the three packages are dedicated to Intel Communication Collective Libraries for Distributed. Currently, Intel GPU only plans to support Linux.